### PR TITLE
endpoints front end

### PR DESCRIPTION
**Pull request by JonasVervloet**
New endpoints created for the frontend:

/cities
/lists
/candidates

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -78,92 +78,133 @@ async def r_get(req, name):
...
-        {
-            'succes': 'true',
-            'result': result_json
-        }
-    )
+    candidates = helper_sparql.get_lblod_candidates(list_url)
+    return response.text('Work in progress...')
+
+
+# @app.route('/candidates', methods=['GET'])
```
Comment (juliavdkris) [inline src/main.py]: You can remove all the commented out stuff. If we ever need it, we can just get it back from the git history

**Review by juliavdkris — CHANGES_REQUESTED** [review]
You can remove the commented out stuff, but other than that looks great!

File: src/main.py (hunk 1 above)
Comment (JonasVervloet) [reply src/main.py]: Yeah, I forgot to push two commits

**Review by juliavdkris — APPROVED** [review]
Awesome, looks good to me!

**Review by juliavdkris — APPROVED** [review]
LGTM

**Pull request merged**